Framework/Geometry/src/Instrument.cpp

```diff
@@ -816,7 +816,12 @@ std::vector<detid_t> Instrument::getMonitors() const {
  */
 void Instrument::getBoundingBox(BoundingBox &assemblyBox) const {
```

Comment:
Would it be too much work to change the signature of this method to  `BoundingBox Instrument::getBoundingBox() const` ? That feels like a better signature for this.

Reply:
I was trying not to change the public API of `Instrument`, though you will see for `API::ComponentInfo` I am returning by value via `BoundingBox API::ComponentInfo::boundingBox() const`. Since `Instrument` is being phased out, I wanted to minimise changes there. Is there a particular reason you want this changed?

Reply:
No particular reason. I must admit I'd forgotten that this class wasn't going to survive the phase out so it was purely a cleanup argument.